src/System.Threading/src/System.Threading.csproj

```diff
@@ -6,7 +6,14 @@
     <AssemblyName>System.Threading</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <GenFacadesIgnoreMissingTypes Condition="'$(TargetGroup)'=='uapaot'">true</GenFacadesIgnoreMissingTypes>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetGroup)' == 'uapaot'">
+    <GenFacadesIgnoreMissingTypes>true</GenFacadesIgnoreMissingTypes>
+    <!-- Suppress warning for LockRecursionPolicy type conflicts between this assembly and System.Private.CoreLib. -->
+    <!-- Remove when LockRecursionPolicy is removed from System.Private.CoreLib. -->
+    <NoWarn>0436</NoWarn>
+    <!-- Remove when CDSsyncETWBCLProvider is supported -->
+    <DefineConstants>$(DefineConstants);UAPAOT</DefineConstants>
```

Comment:
Nit: This should be lowercase (https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md at the bottom)